### PR TITLE
318 hermes console opensource

### PR DESCRIPTION
**Pull request by adamdubiel**
Includes

* Hermes Console files
* Hermes Console packaging & run scripts
* Hermes Console in Vagrant

### CODE REVIEW

**Comment by adamdubiel** [comment]
this solves #318

File: README.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ Hermes is an asynchronous message broker built on top of [Kafka](http://kafka.ap
 We provide reliable, fault tolerant REST interface for message publishing and adaptive push
 mechanisms for message sending.
 
-See our 10-minute getting started guide with docker: [Getting started](http://hermes-pubsub.readthedocs.org/en/latest/contents/tutorials/01_getting_started.html)
+See our 10-minute getting started guide with docker: [Getting started](http://hermes-pubsub.readthedocs.org/en/latest/quickstart/)
```
Comment (dankraw) [inline README.md]: let's remove the "with docker" part  :)

File: docs/docs/deployment/deployment.md [hunk 2]
```diff
@@ -24,7 +25,22 @@ performance and easy maintenance, each Hermes module should also be deployed on
 
 ## Requirements
 
-All Hermes modules require **Java 8** to work.
+All Hermes Java modules require **Java 8** to work. Hermes Console has no external dependencies.
+
+## Passing environment variables
+
+All Java modules share the same bundling strategy: Gralde distZips. In order to pass any command line options to
```
Comment (dankraw) [inline docs/docs/deployment/deployment.md]: typo Gralde

File: hermes-console/package.json [hunk 3]
```diff
@@ -0,0 +1,30 @@
+{
+  "name": "hermes-console",
+  "version": "1.0.0",
```
Comment (dankraw) [inline hermes-console/package.json]: maybe somehow we could stick to hermes version here?

**Comment by dankraw** [comment]
Great! :+1:

**Comment by adamdubiel** [comment]
I added integration with [Consul](http://consul.io) for Hermes Management discovery.

**Comment by druminski** [comment]
:+1:

**Comment by adamdubiel** [comment]
merging - travis-ci fix is in progress, did build locally

**Pull request merged**